test/033_wsteth.ts

```diff
@@ -0,0 +1,248 @@
+import { SignerWithAddress } from '@nomiclabs/hardhat-ethers/dist/src/signer-with-address'
```

Comment:
Ladle tests are in the `06*` and `07*` group, and here I think we will be testing the wstEth wrapping module, so I would choose the `074_ladle_wsteth.ts` file name.

I think we will need to create subfolders in the `test` folder, in a separate PR.